lib/PDF/API2.pm

```diff
@@ -1946,6 +1946,9 @@ extension.  If you're using a font with an atypical extension, you can set
 C<format> to one of C<truetype> (TrueType or OpenType), C<type1> (PostScript
 Type 1), or C<bitmap> (Adobe Bitmap).
 
+Setting format to C<truetype> is mandatory if the C<$name> argument is a
+C<Font::TTF::Font> object.
```

Comment:
Can you remove this requirement?  It seems like this is something that can be inferred.